client/src/i18n/en/report.json

```diff
@@ -141,6 +141,10 @@
       "DESCRIPTION": "This report shows the current stock value for each inventory in a depot",
       "EXCLUDE_INVENTORIES_ZERO_VALUE": "Exclude inventories with zero stock value"
     },
+    "STOCK_CONSUMPTION_GRAPH_REPORT" : {
+      "TITLE" : "Stock Consumption's graph report",
+      "DESCRIPTION" : "This report graphically shows the consumption of stock during a period for a depot or inventory"
```

Comment:
Does it show it by quantity consumed?  Or value of the stock?  Or what is it referencing?

Reply:
Can you add in text about what is being grouped to this description?